src/pages/practice/styles/PracticePage.css

```
@@ -636,6 +655,128 @@
+  cursor: pointer;
+}
+
+.practice-style-card.is-selected .practice-style-card__speaker {
+  color: #22b8a7;
+}
+
+.practice-style-card__speaker svg {
+  width: 100%;
+  height: 100%;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)